config/prow/cluster/build_kubernetes-external-secrets_customresource.yaml

```diff
@@ -13,3 +13,16 @@ spec:
   - key: azure-cred
     name: credentials
     version: latest
+---
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: gcp-secrets-store-cred
```

Comment:
Just to confirm, this name will be used for the synced secret name as well?

Reply:
That was my understanding from https://github.com/kubernetes/test-infra/blob/master/prow/prow_secrets.md#usage-prow-clients

>   name: <my-precious-secret-kes-name>    # name of the k8s external secret and the k8s secret